src/pages/_app.tsx

```diff
@@ -23,8 +24,10 @@ const Application = ({ Component, pageProps }: AppProps) => {
       <ThemeProvider theme={theme}>
         <QueryClientProvider client={queryClient}>
           <AuthenticationProvider>
-            <Toast />
-            <Component {...pageProps} />
+            <ToastContextProvider>
+              <Toast />
```

Comment:
you have already passed Toast inside to `ToastContextProvider`. there is no need for this component here